convert_to_base64_MOD.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+  name: 'Convert to Base64',
+  section: 'Other Stuff',
+
+  subtitle (data) {
+    return 'Convert To Base64'
+  },
+
+  variableStorage (data, varType) {
+    if (parseInt(data.storage) !== varType) return
+    return ([data.varName, 'Convert To Base64'])
```

Comment:
It would be outputing a string, not a "Convert To Base64".
```suggestion
    return ([data.varName, 'String'])
```